omiclearn/omic_learn.py

```diff
@@ -499,8 +500,8 @@ def OmicLearn_Main():
             - Using the following features: **Class 0 `{state.class_0}`, Class 1 `{state.class_1}`**.
             - Using classifier **`{state.classifier}`**.
             - Using a total of  **`{len(state.features)}`** features.
-            - Note that OmicLearn is intended to be an exploratory tool to assess the performance of algorithms,
-                rather than providing a classification model for production.
+            - ⚠️ Warning: OmicLearn is intended to be an exploratory tool to assess the performance of algorithms,
+                rather than providing a classification model for production. Please check our [recommendations](https://omiclearn.readthedocs.io/en/latest/recommendations) - page for potential pitfalls and interpret performance metrics accordingly.
```

Comment:
I think the link should include `.html` at the end, be like `https://omiclearn.readthedocs.io/en/latest/recommendations.html` once I check the other pages: https://omiclearn.readthedocs.io/en/latest/VERSION-HISTORY.html